package.json

```diff
@@ -62,6 +62,7 @@
     "nan": "^2.6.2",
     "prebuild-install": "^2.4.1",
     "promirepl": "^1.0.1",
+    "prompt-list": "^3.1.2",
```

Comment:
This change does require a new dependency. I don't know if this will be contentious or not. Please let me know.